listenbrainz/webserver/views/atom.py

```diff
@@ -171,12 +218,14 @@ def get_listens(user_name):
 
         content = render_template(
             "atom/listens.html",
-            user_page_url=_external_url_for("user.index", path="", user_name=user_name),
+            user_page_url=_external_url_for(
+                "user.index", path="", user_name=user_name),
             user_name=user_name,
             recording_mb_page_base_url="https://musicbrainz.org/recording/",
             track_name=track_name,
             recording_mbid=recording_mbid,
```

Comment:
Unable to comment a few lines above so doing it here, I think the listens should fallback to using mapped mbid data if the additional_info doesn't have the required fields? But not a blocker for merging, we can do this later as well.